dev_course/dl2/02b_initializing.ipynb

```diff
@@ -6,7 +6,8 @@
    "metadata": {},
```

Comment:
<p>I think the above 2 Jeremy will object to, since you're changing the code - i.e. it won't match the video. On the other hand, this part wasn't really shown, so it's probably OK.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_docs/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>